Packs/XForceExchange/Integrations/XFE_v2/XFE_v2.py

```diff
@@ -379,13 +379,18 @@ def file_command(client: Client, args: Dict[str, str]) -> Tuple[str, dict, Any]:
 
         file_key = f'XFE.{outputPaths["file"]}'
 
-        hash_info = {**report['origins'], 'Family': report['family'], 'FamilyMembers': report['familyMembers']}
+        report_data = report['origins'].get('external', {})
+        family_value = report_data.get('family')
+
+        hash_info = {**report['origins'], 'Family': ', '.join(family_value),

```

Comment:
why did you switch this?
```suggestion
        hash_info = {**report['origins'], 'Family': family_value,

```

Reply:
family value is array, you can see example in the PR description

Reply:
Leave it as list